src/main/java/core/basesyntax/MyNode.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class MyNode<T> {
+    private T value;
+    private MyNode<T> prev;
+    private MyNode<T> next;
+
+    MyNode(T x) {
+        value = x;
+    }
```

Comment:
The constructor should initialize all fields (`value`, `prev`, `next`) as per the checklist: 'When creating a new `Node<T>`, pass all the fields to the `Node` constructor.'